File: java-sfn-unit-test/src/main/java/com/sfnut/myapp/parent/Step40_Parallel_LambdaB.java
package com.sfnut.myapp.parent;

import com.sfnut.sfn.lambda.SfnLambda;

public class Step40_Parallel_LambdaB extends SfnLambda<ParentPayload> {
    @Override
    public ParentPayload execute(ParentPayload payload) {
        payload.setParallelB("Bar");
        return payload;
    }
}

File: myapp/parent/Step40_Parallel_LambdaB.cs
using sfn_ut.sfn.lambda;

namespace sfn_ut.myapp.parent;

public class Step40_Parallel_LambdaB : SfnLambda<ParentPayload> {
  public override ParentPayload Execute(ParentPayload payload) {
    payload.ParallelB = "Bar";
    return payload;
  }
}
